src/components/TodayWaterList/TodayListIcons.jsx

```diff
@@ -0,0 +1,93 @@
+  switch (id) {
+    case 'delete-icon': {
+      svg = (
+        <svg
+          className="delete-icon"
+          version="1.1"
+          xmlns="http://www.w3.org/2000/svg"
+          width={width}
+          height={height}
+          viewBox="0 0 1024 1024"
+        >
+          <title>delete</title>
+          <path
+            fill="none"
+            stroke-linejoin="round"
+            stroke-linecap="round"
+            stroke-miterlimit="4"
+            stroke-width="42.6667"
+            d="M628.907 384l-14.763 384M409.856 768l-14.763-384M820.395 247.040c14.592 2.219 29.099 4.565 43.605 7.083M820.395 247.040l-45.568 592.341c-3.952 49.747-45.294 88.619-95.716 88.619-0.010 0-0.020 0-0.030 0h-334.164c-0.008 0-0.018 0-0.028 0-50.422 0-91.764-38.872-95.694-88.283l-0.021-0.336-45.568-592.341M820.395 247.040c-38.791-6.358-89.724-12.366-141.193-16.476l-7.202-0.463M203.605 247.040c-25.242 3.599-40.074 6.065-54.844 8.706l11.239-1.666M203.605 247.040c38.789-6.358 89.722-12.366 141.19-16.476l7.205-0.463M672 230.101v-39.083c0-50.347-38.827-92.331-89.173-93.909-21.11-0.721-45.921-1.132-70.827-1.132-24.908 0-49.719 0.411-74.428 1.226l3.602-0.094c-50.347 1.579-89.173 43.605-89.173 93.909v39.083M672 230.101c-47.901-3.932-103.689-6.173-160-6.173-56.31 0-112.097 2.241-167.276 6.64l7.276-0.466"
+          ></path>
+        </svg>
+      );
+      break;
+    }
+    case 'edit-icon': {
+      svg = (
+        <svg
+          version="1.1"
+          xmlns="http://www.w3.org/2000/svg"
+          width={width}
+          height={height}
+          viewBox="0 0 1024 1024"
+        >
+          <title>edit</title>
+          <path
+            className="edit-icon"
+            fill="none"
+            stroke-linejoin="round"
+            stroke-linecap="round"
+            stroke-miterlimit="4"
+            stroke-width="42.6667"
+            d="M719.445 191.445l71.979-72.021c14.479-14.479 34.482-23.435 56.576-23.435 44.189 0 80.011 35.822 80.011 80.011 0 22.094-8.955 42.097-23.435 56.576l-453.077 453.077c-21.936 21.924-49.153 38.567-79.579 47.856l-1.36 0.357-114.56 34.133 34.133-114.56c9.647-31.785 26.289-59.002 48.214-80.94l381.098-381.055zM719.445 191.445l112.555 112.555M768 597.333v202.667c0 53.019-42.981 96-96 96v0h-448c-53.019 0-96-42.981-96-96v0-448c0-53.019 42.981-96 96-96v0h202.667"
+          ></path>
+        </svg>
+      );
+      break;
+    }
+    case 'cup-icon': {
+      svg = (
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="36"
+          height="36"
+          fill="none"
+        >
+          <path d="M6.261 3.13 8.4 32.79v.08h18.412l2.139-29.646V3.13H6.26ZM26.056 32.04H9.162L7.558 9.824h.882l1.45 19.578c.009.103.055.199.13.27.074.07.172.11.274.11h.034a.401.401 0 0 0 .356-.279.414.414 0 0 0 .02-.159L9.252 9.824H27.66l-1.604 22.215Zm1.66-23.046H9.19l-.318-4.328a.423.423 0 0 0-.137-.274.412.412 0 0 0-.284-.107.413.413 0 0 0-.276.146.425.425 0 0 0-.1.299l.319 4.27h-.893l-.364-5.043h20.944l-.364 5.037Z" />
```

Comment:
do icon like this 
https://github.com/imiryna/Chat-Box/blob/2ae4e82c06cb1873e927e41c855d8998071798d6/src/components/chatTopic/chatTopic.styled.jsx#L2